### PR TITLE
Responsiveness of images in blogs increased issue number #499

### LINKED ISSUE #499: Extend and increase responsiveness of images in the Blogs and video sections
### Expected results
image

### Steps to reproduce the issue
Change the responsiveness labels used  
## Would you like to work on this

Doing that

### PR DESCRIPTION
**Pull request by sandeepyadav097**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request
Fixes #499 
Added and improved responsive labels
preview-
image

Actual-
image

### CODE REVIEW

**Comment by anshumanv** [comment]
I'd like to request a preview link to test the changes you made. :smile:

**Comment by sandeepyadav097** [comment]
The link is http://sandeepyadav.tech/gci17.fossasia.org/

File: index.html [hunk 1]
```diff
@@ -415,8 +415,8 @@ <h4 class="uppercase mb16">Videos</h4>
 			{% if modulo == '1' %}
 		<div class="row">
 			{% endif %}
-			<div class="col-md-3">
-				<a href="{{video.url}}" target="_blank">
+		    <div class="col-md-4 col-lg-3 col-sm-4 col-xs-6">
```
Comment (anshumanv) [inline index.html]: Indent this line properly.

**Review by anshumanv — APPROVED** [review]
Works well.

**Comment by sandeepyadav097** [comment]
Merge them

**Review by NaviRocker — CHANGES_REQUESTED** [review]
@sandeepyadav097 Try to Squash your commits. It would be Awesome.

**Comment by sandeepyadav097** [comment]
Can you tell me How? @naveentherocker

**Comment by NaviRocker** [comment]
Try this @sandeepyadav097 
`git reset --soft HEAD~5`
`git commit -m "<Your commit message>"`
`git push -f origin gh-pages`

**Comment by NaviRocker** [comment]
And try to mention the Issue that you are working on.

**Review by NaviRocker — APPROVED** [review]
Please Mention `Fixes #499` in the PR Body.

File: index.html [hunk 2]
```diff
@@ -303,21 +303,40 @@ <h6 class="uppercase mb16">Technologies used: Kotlin, Java, Shell Script, Python
...
 <section>
     <div class="container">
-        <div class="row">
+        <div class="row" style="display:table">
             <div class="col-sm-4">
                 <h2 class="number">Phimp.me</h2>
                 <p>
                     Do you have many different apps in your Android Phone for the purpose of clicking, editing and sharing images? Hey guess what, now you don't need all of those because Phimp.me is here, with all the features packaged into one. So go and clean your phone memory to save your space for other useful purposes.
                 </p>
+
```
Comment (NaviRocker) [inline index.html]: Remove unwanted lines Please.

File: index.html [hunk 3]
```diff
@@ -303,21 +303,40 @@ <h6 class="uppercase mb16">Technologies used: Kotlin, Java, Shell Script, Python
         </div>
     </div>
 </section>
+
```
Comment (NaviRocker) [inline index.html]: Remove these

**Review by NaviRocker — CHANGES_REQUESTED** [review]
There are many unwanted lines. Try to remove them

File: index.html (hunk 3 above)
Comment (sandeepyadav097) [reply index.html]: I haven't made any changes on those lines I guess

**Comment by sandeepyadav097** [comment]
@anshumanv who will merge these

**Comment by anshumanv** [comment]
> @anshumanv who will merge these

Some mentor after this has gained enough approves.

**Review by ParthS007 — COMMENTED** [review]
@sandeepyadav097  Why have you changed other files, Please make changes which are related to the issue.

**Review by ParthS007 — COMMENTED** [review]
This is actual site in `Samsung Galaxy Note 3`
image
This is in your Preview link 
image


Its just a example, Please make necassary changes so that names do not overlap with blog title.

**Comment by sandeepyadav097** [comment]
@ParthS007  The images look overly stretched on iPad. What do you suggest me to close the PR and restore it ??


The images are highly stretched for iPad.

**Pull request closed without merging**